pkg/minikube/update/update_test.go

```diff
@@ -1,183 +0,0 @@
-/*
```

Comment:
We are removing this because all test code is actually commented out and we are not using update functionality right now.